index.js

```diff
@@ -5,8 +5,10 @@ const fs = require('fs')
 const path = require('path')
 const readDirectory = promisify(fs.readdir)
 const pkgUp = require('pkg-up')
+const semver = require('semver')
 
 const typescriptSupport = Symbol.for('ts-node.register.instance') in process
+const moduleSupport = semver.satisfies(process.version, '>=14')
```

Comment:
```suggestion
const moduleSupport = semver.satisfies(semver.coerce(process.version), '>=14')
```

Should it be that instead? I believe that will allow for Node 15, 16, .... N.